test/aqua/test_amplitude_estimation_circuitfactory.py

```diff
@@ -563,9 +562,7 @@ def test_conditional_value_at_risk(self, simulator):
         # n_s = WeightedSumOperator.get_required_sum_qubits(lgd)
 
         # create circuit factory (add Z qubits with weight/loss 0)
-        warnings.filterwarnings('ignore', category=DeprecationWarning)
         agg = WeightedSumOperator(n_z + k_l, [0] * n_z + lgd)
```

Comment:
WeightedSumOperator is deprecated, but I guess this was wrapped in a larger filter.

Reply:
Yes, there was already a deprecation filter in the setup method of that test class and the two statements that I removed here were causing the filter to be changed before time and affecting the the methods.